start.py

```diff
@@ -55,12 +56,14 @@ def gen_config():
 
     conf = conf + "\nREDIS_PORT = " + str(rport)
     conf = conf + "\nREDIS_DB = " + str(rdb)
+
     if rpwd:
         conf = conf + "\nREDIS_PASSWD = \"" + rpwd + "\""
     else:
-        conf = conf + "\nREDIS_PASSWD = None"
-    conf = conf + "\n\n# Telegram Bot API token from t.me/BotFather"
-    conf = conf + "\nBOT_TOKEN = \"" + token + "\"\n"
+        conf += "\nREDIS_PASSWD = None"
```

Comment:
You asked me to change from REDIS_PASSWD to REDIS_PASSWORD
I've changed it, you don't